packages/flutter_tools/lib/src/version.dart

```diff
@@ -236,21 +236,38 @@ class FlutterVersion {
     final Duration frameworkAge = _clock.now().difference(localFrameworkCommitDate);
```

Comment:
Let's make kVersionAgeConsideredUpToDate be 5 weeks instead of 4, since we release betas every 4 weeks. No need to push people to update too aggressively.